css/index.css

```diff
@@ -0,0 +1,289 @@
+
+body {
+    font-family: 'Manrope', sans-serif;
+    font-weight: bold;
+    color: #555e6c;
+    font-size: 14px;
+}
+
+.container {
+    margin-left: 40px;
+    margin-right: 40px;
+    margin-bottom: 40px;
+}
+
+ul {
+    background-color: blue;
+    list-style-type: none;
+}
+
+li {
+    float: left;
+    padding: 8px 10px;
+}
+
+hr.new1 {
+    height: 1px;
+    border: 0;
+    border-top: 1px solid #f4f3f6;
+    width: 100%;
+}
+
+.nav { 
+    float: left;
+    color: #ccd1d7;
+    font-weight: bold;
+    font-family: 'Manrope', sans-serif;
+}
+
+.patch { 
+    font-weight: bold;
+    color:#1a2939;
+}
+
+.depo {
+    color:#47a8d1;
+}
+
+.img {
+    margin-left: 550px;
+    margin-top:-20px;
+    overflow: hidden;
+}
+
+img {
+    border-radius: 25px;
+    height: 25px;
+    width: 25px;
+}
+
+
+.btn-primary {
+    background-color:#4ecb73;
+    color: #e5fdf9;
+    height: 45px;
+    width: 145px;
+    border-radius: 8px;
+    float: right;
+    margin-left: 868px;
+    font-weight: bold;
+    border: none;
+    margin-bottom: 15px
+}
+
+.btn-depo {
+    float: left;
+}
+
+.head3 {
+    color: #313e4c;
+    margin-top: 5px;
+    font-weight:normal;
+}
+
+.table {
+    text-align: left;
+}
+
+table {
+    border-collapse: collapse;
+    height: 100%;
+    width: 100%;
+}
+
+th {
+    padding: 10px 0px;
+    background-color: #f9fbfc;
+    font-size: 10px;
+    color:#9aa3b0;
+    letter-spacing: 1px;
+}
+
+td {
+    padding-top: 10px;
+    padding-left: 0px;
+}
+
+.line {
+    border-bottom: 1px solid #f4f3f6;
+}
+
+th.row1 {
```

Comment:
@indalgit same mistake

Reply:
@indalgit you can break this class into it 3 parts: e.g **pr-200** (i.e padding-right), **pl-8** and separate class for color. And finally you can give all these 3 class names to row1 element instead of row1.